tests/cassettes/InferenceClientVCRTest.test_text_classification.yaml

```
@@ -0,0 +1,48 @@
+interactions:
+- request:
+    body: '{"inputs": ["I like you", "I love you."]}'
```

```suggestion
    body: '{"inputs": ["I like you"]}'
```

should be only 1 sample now